hypothesis-python/src/hypothesis/extra/array_api.py

```diff
@@ -62,9 +62,7 @@
 from hypothesis.strategies._internal.strategies import check_strategy
 from hypothesis.strategies._internal.utils import defines_strategy
 
-__all__ = [
-    "make_strategies_namespace",
-]
+__all__ = ["make_strategies_namespace"]
```

Comment:
Drop this diff and the release notes, and I think we're ready to merge!

Reply:
Note to self: stop being pedantic about non-consequential changes to minimise the diffs :sweat_smile: